lego/apps/users/constants.py

```diff
@@ -142,3 +142,12 @@ def values(cls) -> list[str]:
     (LIGHT_THEME, LIGHT_THEME),
     (DARK_THEME, DARK_THEME),
 )
+
+
+LATE_PRESENCE_PENALTY_WEIGHT = 1
```

Comment:
Would be nice to keep all the penalty weights togheter
```suggestion
class PENALTY_WEIGHTS(models.IntegerChoices):
	PRESENCE = 1
	...
```

Reply:
Seems like maybe you didn't remove the other constant, but that can be fixed later

Reply:
oops